app/presenters/tree_builder_tenants.rb

```diff
@@ -23,7 +23,7 @@ def tree_init_options
   end
 
   def tenant_tree_or_generic
-    if @ansible_playbook
+    if @template
```

Comment:
I wonder if we name the parameter something more meaningful like "show_tenant_tree" ?  Even changing to something more generic like template doesn't tell me what is actually going on.